src/seedu/addressbook/Main.java

```diff
@@ -103,8 +103,10 @@ private void recordResult(CommandResult result) {
      * @param command user command
      * @return result of the command
      */
-    private CommandResult executeCommand(Command command)  {
+    private CommandResult executeCommand(Command command, StorageFile storage)  {
         try {
+        	//check the address book before executing the command.
+        	this.addressBook=storage.load();
```

Comment:
The task here asking you to use Exception to handle missing file when you call save method. Please check how other student complete this task, e.g. #591.